content/scout/policy/_index.md

```diff
@@ -162,3 +163,31 @@ building with attestations, see
 > Docker Scout is currently unable to discern the difference between using
 > `scratch` as a base image and having no base image provenance. As a result,
 > images based on `scratch` always fail the Supply chain attestations policy.
+
+### Quality gates passed
+
+The Quality gates passed policy builds on the [SonarQube
+integration](../integrations/code-quality/sonarqube.md) to assess the quality
+of your source code. This policy works by ingesting the SonarQube code analysis
+results into Docker Scout.
+
+You define the criteria for this policy using SonarQube's [quality
+gates](https://docs.sonarsource.com/sonarqube/latest/user-guide/quality-gates/).
+SonarQube evaluates your source code against the quality gates you've defined
+in SonarQube. Docker Scout surfaces the SonarQube assessment as a Docker Scout
+policy.
+
+Docker Scout uses [provenance](../../build/attestations/slsa-provenance.md)
+attestations to link SonarQube analyses with container images. In addition to
+enabling the SonarQube integration, you must also make sure that your images
+have provenance attestations with information about the Git repository
+containing the source code of the image.
+
```

Comment:
I think it'd be beneficial to add this image (or a similar one) that shows how the Git revision is the link between the container image and the SQ analysis results:

![image](https://github.com/docker/docs/assets/15997951/a84a6186-36f0-4424-94d2-53a652b77c57)